app/src/main/java/com/example/dine_and_donate/Models/Restaurant.java

```diff
@@ -0,0 +1,17 @@
+package com.example.dine_and_donate.Models;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+public class Restaurant {
+
+    public static String format(JSONObject object) throws JSONException {
+        StringBuilder sb = new StringBuilder();
+        sb.append(object.getString("name") + "\n");
```

Comment:
The whole point of StringBuilder is to avoid concatenating strings (since Strings are immutable, you create a new one every time you do "string1" + "string2"). So instead of doing "+ "\n"", remove that and on another line do "sb.append("\n"))".